### PR TITLE
hystogram function with opening file 

### PR DESCRIPTION
**Pull request by Tristan1649**
Function of hystogram using Tensorflow methods and include creating file.

### CODE REVIEW

**Comment by Tristan1649** [comment]
> Function of hystogram using Tensorflow methods and include creating file.

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hello @Tristan1649 
Thanks for contributing to Ivy😊

Would you explain more about this function you're implementing also if you're getting it hard to understand the contribution I would recommend you first check the [Contribution guide](https://lets-unify.ai/ivy/contributing.html)?

Thanks and happy coding🙂

**Comment by hirwa-nshuti** [comment]
> > Hello @Tristan1649 Thanks for contributing to Ivy😊
> > Would you explain more about this function you're implementing also if you're getting it hard to understand the contribution I would recommend you first check the [Contribution guide](https://lets-unify.ai/ivy/contributing.html)?
> > Thanks and happy coding🙂
> 
> Hi @hirwa-nshuti Nice to meet you. I need sometime to fixi it because i have some problems wih it. Can we contact in discord?

Sure feel free to text me😊

**Comment by Tristan1649** [comment]
> > > Hello @Tristan1649 Thanks for contributing to Ivy😊
> > > Would you explain more about this function you're implementing also if you're getting it hard to understand the contribution I would recommend you first check the [Contribution guide](https://lets-unify.ai/ivy/contributing.html)?
> > > Thanks and happy coding🙂
> > 
> > 
> > Hi @hirwa-nshuti Nice to meet you. I need sometime to fixi it because i have some problems wih it. Can we contact in discord?
> 
> Sure feel free to text me😊

Hi Felix! how are you? About function. I took a function from Tensorflow. Main idea in 1)taking information from file(xls) 2) proceed it 3) visualisation at hystogram graphic.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Tristan1649** [comment]
to tenzorlow flow adding nupmy, jax, pytorch

File: ivy/functional/backends/torch/experimental/statistical.py [hunk 1]
```diff
@@ -118,3 +118,34 @@ def corrcoef(
         xarr = xarr.T if not rowvar else xarr
 
     return torch.corrcoef(xarr)
+
+
+
+#torch
+@staticmethod
+def histtorch(self: ivy.Array,
```
Comment (hirwa-nshuti) [inline ivy/functional/backends/torch/experimental/statistical.py:126]: For torch backend we should be accepting `torch.Tensor` not `ivy.Array`

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @Tristan1649 
Add the function to the functional side of the Experimental API,  also referring to documentation from both numpy and torch you don't need to print anything what is needed is to calculate the histograms of input tensors/arrays

https://numpy.org/doc/stable/reference/generated/numpy.histogram.html

https://pytorch.org/docs/stable/generated/torch.histogram.html

Also, Each backend is passed its native arrays/tensors as inputs not `ivy.Array`

Thanks😊

**Comment by Tristan1649** [comment]
update

**Comment by Tristan1649** [comment]
update

**Comment by Tristan1649** [comment]
Hi @hirwa-nshuti . I accidiently close issue. Can we contact?
